_data/apis.yml

```diff
@@ -63,7 +63,7 @@
     logo: https://telematici.agenziaentrate.gov.it/resources/img/AgenziaEntrate_logo_152.png
   tags: []
   language: [it]
-  slug: agenzia-entrate-corrispettivi-dispos
+  slug: dispositivi-corrispettivi
```

Comment:
imho retaining the api provider may help disambiguating the same api from different providers.